projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-app-media.php

```diff
@@ -124,17 +124,20 @@ public function get_media( \WP_REST_Request $request ) {
 		$params = $request->get_params();
 		$number = $params['number'];
 
-		$query_args  = array(
+		$query_args = array(
 			'post_type'   => 'attachment',
 			'post_status' => 'inherit',
 			'number'      => $number,
 			'date_query'  => array(
 				'after' => gmdate( DATE_RSS, intval( $params['after'] ) ),
 			),
-			'paged'       => $params['page_handle'],
 			'author'      => get_current_user_id(),
 			'orderby'     => 'date',
 		);
+		if ( isset( $params['page_handle'] ) ) {
+			// No default value.
+			$query_args['paged'] = $params['page_handle'];
+		}
```

Comment:
```suggestion
			'paged'      => $params['page_handle'] ?? 0,
			'author'      => get_current_user_id(),
			'orderby'     => 'date',
		);
```
This is just a nitpick since the code works fine, but I wonder if we could just default to 0 or null in case it's not set. 
Seems it'll be used [here](https://github.com/Automattic/jetpack/pull/43907/files#diff-cb1ee88c67216d468dde0b2b17a2a9c99fa795c958a5193ea9946cee7f3df9bbR163) and it's set to null no matter what since it's a parameter in [WP_Query](https://developer.wordpress.org/reference/classes/wp_query/#pagination-parameters).

Reply:
Thanks for taking a look!

> it's set to null no matter what since it's a parameter in [WP_Query](https://developer.wordpress.org/reference/classes/wp_query/#pagination-parameters).

I decided to explicitly avoid creating a default because `WP_Query` will define the default, and also because we do not declare a default when registering the rest route here: https://github.com/Automattic/jetpack/blob/678a1ed3527acb30647e08bdfd7cecbf1592a290/projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-app-media.php#L46 .

If we did, it would have made sense to me to fallback to a default. That said, I'm open to change this of course if it makes more sense, but just wanted to share my reasoning.

Reply:
> I decided to explicitly avoid creating a default because `WP_Query` will define the default, and also because we do not declare a default when registering the rest route here:
> 
> https://github.com/Automattic/jetpack/blob/678a1ed3527acb30647e08bdfd7cecbf1592a290/projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-app-media.php#L46
> 
> .
> If we did, it would have made sense to me to fallback to a default. That said, I'm open to change this of course if it makes more sense, but just wanted to share my reasoning.

Right, I didn't realize about the param registration. Shouldn't it make more sense then to have a default of 0 to `page_handle`?

Reply:
Within the registration itself? 🤔 Yes that also makes sense, I just don't know if there was a reason to leave it out. From my testing though, both before and after the change, the `paged` default is returning as 1. That is also what we default to elsewhere in the codebase from what I can see ([example](https://github.com/Automattic/jetpack/blob/e0a7c3ab9dd4ddde107ff8cc297d2c18dbe42c8e/projects/packages/search/src/classic-search/class-classic-search.php#L527)).

Reply:
I added the default of 1 to the registration as well as a fallback here. Also cleaned up the `$number` declaration, which seemed redundant (and included the number default there as well as a fallback): cfd795be2ea8997aee1633a62f157599a332c7dd

Reply:
Updated again after our Slack chat - e27fd39a6e7ce1a9d32619f7f26e0fcfdb7a1071 - as the defaults when registering the rest route will be used when set.